server/src/routes/api/v1/players3rdPartyRouter.js

```diff
@@ -0,0 +1,25 @@
+import express from "express"
+
+import PlayersClient from "../../../apiClient/PlayersClient.js"
+import PlayerSerializer from "../../../serializers/PlayerSerializer.js"
+
+const playersRouter = new express.Router()
+
+playersRouter.get("/", (req, res) => {
+  const playerName = req.query.playerName
+
+  PlayersClient.getPlayer(playerName).then((data) => {
+    if (data.error) {
+      console.log(`Error from balldontlie api: ${data.error}`)
+    } else {
+      const parsedResponse = JSON.parse(data)
+      const player = PlayerSerializer.getSummary(parsedResponse.data[0])
+      res
+        .set({ "Content-Type": "application/json" })
+        .status(200)
+        .json(player)
+    }
+  })
+})
```

Comment:
I like these, just make sure that if you are going to use `.then` that you also have a `.catch` block for if something goes wrong while talking to the external api. `.data.error` wont cut it if you get a non-200 status code back